pkg/storage/fs/owncloudsql/spaces.go

```diff
@@ -31,12 +31,14 @@ import (
 	"github.com/cs3org/reva/v2/pkg/errtypes"
 	"github.com/cs3org/reva/v2/pkg/storage/fs/owncloudsql/filecache"
 	"github.com/cs3org/reva/v2/pkg/storagespace"
+	"github.com/cs3org/reva/v2/pkg/utils"
 )
 
 // ListStorageSpaces lists storage spaces according to the provided filters
 func (fs *owncloudsqlfs) ListStorageSpaces(ctx context.Context, filter []*provider.ListStorageSpacesRequest_Filter, unrestricted bool) ([]*provider.StorageSpace, error) {
 	var (
 		spaceID = "*"
+		// uid     *userpb.UserId
```

Comment:
Remove comment

Reply:
not yet. we need to implement the user filter in a subsequent pr. this PR just makes listing spaces and navigating files work again. I'm trying to make things work gradually...